### PR TITLE
[RFR] basic site layout

### PR DESCRIPTION
**Pull request by timchepeleff**

### CODE REVIEW

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+$primary-color: #1c3652;
+$primary-color1: #213c69;
```
Comment (dpickett) [inline app/assets/stylesheets/style.scss]: I would consider different variable names. You can only have one primary color. The others might be secondary or tertiary?

File: app/assets/stylesheets/style.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+  i {
+    font-size: 100px;
+  }
+
+  h4 {
+    color: $white;
+    font-size: 12px;
+    font-weight: 400;
+    margin-bottom: 10px;
```
Comment (dpickett) [inline app/assets/stylesheets/style.scss]: px based dimensions do not account for mobile and other form factors. It is best to use `percentages`, `em`, or `rem`

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -6,11 +6,11 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-  
+
   <nav class="top-bar">
     <ul class="title-area">
       <li class="name">
-        <h1><a href="/">GreatWorkspaceReview</a></h1>
+        <h1><a href="/"><%= image_tag "logo.png", size: '210' %></a></h1>
```
Comment (dpickett) [inline app/views/layouts/application.html.erb]: any image tag should have an `alt` attribute for accessibility purposes

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -32,8 +32,56 @@
...
+          <li><a href="#">Link 2</a></li>
+          <li><a href="#">Link 3</a></li>
+          <li><a href="#">Link 4</a></li>
+        </ul>
+      </div>
+
+      <div class="medium-10 columns content">
+        <%= yield %>
+      </div>
+      </div>
```
Comment (vikram7) [inline app/views/layouts/application.html.erb]: This is either a hanging `div` or it's not aligned with it's opening `div` so make sure to check that out.

**Pull request merged**